src/main/java/games/strategy/engine/data/GameDataMemento.java

```diff
@@ -0,0 +1,136 @@
+package games.strategy.engine.data;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Maps;
+
+import games.strategy.util.Version;
+import games.strategy.util.memento.MementoExporter;
+import games.strategy.util.memento.MementoImportException;
+import games.strategy.util.memento.MementoImporter;
+import games.strategy.util.memento.PropertyBagMementoExporter;
+import games.strategy.util.memento.PropertyBagMementoImporter;
+
+/**
+ * Provides factory methods for creating objects that can import mementos to and export mementos from game data.
+ */
+public final class GameDataMemento {
+  @VisibleForTesting
+  static final String SCHEMA_ID = "application/x.triplea.game-data-memento";
+
+  /**
+   * An immutable collection of default memento export options.
+   *
+   * <p>
+   * The key is the option name. The value is the option value.
+   * </p>
+   */
+  public static final Map<ExportOptionName, Object> DEFAULT_EXPORT_OPTIONS_BY_NAME = newDefaultExportOptionsByName();
+
+  private GameDataMemento() {}
+
+  @VisibleForTesting
+  interface PropertyNames {
+    String NAME = "name";
+    String VERSION = "version";
+  }
+
+  private static Map<ExportOptionName, Object> newDefaultExportOptionsByName() {
+    return Maps.immutableEnumMap(ImmutableMap.of(
+        ExportOptionName.EXCLUDE_DELEGATES, false));
```

Comment:
@DanVanAtta I think there is the potential to have other operations here to figure out how to improve data copy speed and reduce size. One thought I had was an 'exclude history' option as the battle calc and AI don't care about game history. Essentially unless we come up with a better way to copy game data for simulation purposes this could expand here.